3b. DataFactory - Basic SQL and Datamodel Overview/Standard Templates for Views, Procedures/Stored_Procedures_MustHaves_DATOUTPUT.md

```diff
@@ -0,0 +1,110 @@
+
+# Stored procedures must-haves | DATAOUTPUT/result - procedures
+
+- Header with information on content of procedure, creation date, author
+- Drop procedure if already exists and create procedure
+- Clean IDs
+- Set nocount on
+- Check if user is an active user
+- Check user rights and provide output according to rights
+- Write API log entry
+- Grant execute to PlanningFactoryUser & PlanningFactoryService
+

```

Comment:
* every DATAOUTPUT Procedure must have the parameters: @Username, @FactoryID,@ProductLineID (to use it with the webclient)
* to call a dataoutput procedure from a webcluster via PowerQuery it must be named as dbo.sx_pf_DATAOUTPUT_Name

Reply:
Yes, I think all things are mentioned